subgraphs/starter-example/README.md

```diff
@@ -0,0 +1,121 @@
+# subgraph-starter-example
+
+This is a starter example for creating a subgraph using the Hedera network. Using the SwirldLabs Hosted TheGraph Service, however you can use any other service that supports TheGraph for deploying the subgraph, or a self hosted service.
+
+It indexes a simple ERC20 (HTS) token transfer events on the Hedera network.
+
+## Getting Started
+
+### 1. Clone the Repository and copy the Starter Example
+
+Change `your-subgraph-name` to the name of your subgraph.
+
+```bash
+# Clone the whole repository
+git clone git@github.com:hashgraph/hedera-the-graph.git
+# Copy the starter example to your subgraph name folder
+cp -r hedera-the-graph/subgraphs/starter-example your-subgraph-name/
+# Remove the whole repository
+rm -rf hedera-the-graph
+# Change directory to your subgraph
+cd your-subgraph-name
+```
+
+
+### 2. Update the Environment Variables
+
+Update the `.env` file with the API endpoint and access token, using the auth-server and using credentials provided by the SwirldLabs team.
+
+```bash
+cp .env.example .env
+```
+
+Your `.env` file should look something like this:
+
+```bash
```

Comment:
It might happen that this text gets out of sync with the `.env` file.
Is it worth having this section if the `.env.example` file is kept up to date?

Reply:
I think it is, I don't expect the `.env` to change that much, since this is a simple example.